packaging/distribution/etc/systemd/system/tremor.service

```diff
@@ -7,7 +7,7 @@ Requires=network-online.target
 [Service]
 User=tremor
 Group=tremor
-ExecStart=/usr/bin/tremor server run --logger-config /etc/tremor/logger.yaml
+ExecStart=/usr/lib/tremor/tremor.service
```

Comment:
do we really wanna name this shellscript `.service` ? this seems confusing to me, as systemd service files already have the `.service` suffix.

Reply:
I'm indifferent to the name, it's used in the `service`, it starts the `service` so I figured `service` is a good name :D  but if you got a better name I'm happy to change it.

Reply:
what about `tremor_systemd_exec_start.sh`? or just `tremor.sh`?

Reply:
changed it to `tremor.sh` :+1: I think I got them all :tm: but if you can look over the files to make sure I didn't missed any renames :D